lib/galaxy/util/watcher.py

```diff
@@ -154,6 +154,18 @@ def _handle(self, event):
                 callback(path=path)
 
 
+class SimpleFileModifiedEventHandler(FileSystemEventHandler):
+    """ Minimalist event handler; when all we need is callback() on file modified """
+    def __init__(self, watcher):
+        self.watcher = watcher
+
+    def on_modified(self, event):
+        path = os.path.abspath(event.src_path)
+        callback = self.watcher.file_callbacks.get(path, None)
```

Comment:
```suggestion
        callback = self.watcher.file_callbacks.get(path)
```
since None is the default. That's just for consistency.

Reply:
Yes. Also here: https://github.com/galaxyproject/galaxy/blob/d02d627aa7e1e8e66923febb15db9d18579bb32c/lib/galaxy/util/watcher.py#L130